src/actions/guides/frontend/html_forms.cr

```diff
@@ -0,0 +1,533 @@
+class Guides::Frontend::HtmlForms < GuideAction
+  guide_route "/frontend/html-forms"
+
+  def self.title
+    "HTML Forms"
+  end
+
+  def markdown : String
+    <<-MD
+    ### Working with Lucky's HTML builder
```

Comment:
Thoughts on "Overview" so it doesn't take as much space in the table of contents?

Reply:
😂 Yes, that's much better. I had a lack of words on that.